lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +254,132 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.article_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.article_url, self.article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        text_elements = article_soup.find_all(
+            "div", class_="article__paragraph article__paragraph_second"
+        )
+        text_list = [element.get_text(strip=True) for element in text_elements]
+        text = "\n".join(text_list)
+        self.article.text = text
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        title_tag = article_soup.find_all('h1', {'class': 'article__title'})[0]
+        if title_tag:
+            title = title_tag.get_text(strip=True)
+        else:
+            title =  "NOT FOUND"
+        self.article.title = title
+
+        author_tag = article_soup.find_all('p', {'class': 'article__prepared'}) or \
+                     article_soup.find_all('b')
+        if author_tag:
+            authors = [author_tag[0].get_text(strip=True)]
+        else:
+            authors = ["NOT FOUND"]
+        self.article.author = authors
+
+        topic_tag = article_soup.find('div', {'class': 'article__category'}).find('a')
+        if topic_tag:
+            topic = topic_tag.get_text(strip=True)
+        else:
+            topic = "NOT FOUND"
+        if isinstance(topic, list):
+            topic = topic[0]
+        self.article.topics = topic
+
+
+        date_tag = article_soup.find('div', {'class': 'article__date'})
+        if date_tag:
+            date_str = date_tag.get_text(strip=True)
+        else:
+            date_str = "NOT FOUND"
+        date = self.unify_date_format(date_str)
+        if date:
+            self.article.date = date
+        else:
+            self.article.date = datetime.datetime.now()
```

Comment:
why? it is wrong information - you are scrapping web resource and date is critical. show examples when date is not available